src/algoliasearch.helper.js

```diff
@@ -145,7 +145,7 @@ AlgoliaSearchHelper.prototype.search = function() {
  * }
  */
 AlgoliaSearchHelper.prototype.searchOnce = function(options, cb) {
-  var tempState = this.state.setQueryParameters(options);
+  var tempState = this.state.setQueryParameters(options || {});
```

Comment:
If options is not set, then maybe we should just do `options ? this.state.setQueryParameters(options) : this.state`?